packages/3d-web-experience-client/src/Networked3dWebExperienceClient.ts

```diff
@@ -52,13 +52,32 @@ type MMLDocumentConfiguration = {
   };
 };
 
+export type EnvironmentConfiguration = {
```

Comment:
We should move this to the `core` library (in the `composer.ts` file) minus the `enableTweakPane` field so that the core doesn't depend on this high level library.